packages/bee-q/src/components/toast/_storybook/bq-toast.mdx

```diff
@@ -0,0 +1,13 @@
+import { ArgTypes, Stories, Title, Subtitle } from '@storybook/addon-docs';
+
+<Title>Toast</Title>
+
+<Subtitle>Usage</Subtitle>
+
+<Subtitle>👍 When to use</Subtitle>
+
+<Title>Properties</Title>
+
+<ArgTypes of="bq-toast" />
+
+<Stories title="Usage examples" />
```

Comment:
I think we can remove the Stories from the Overview docs, the examples are right above that 🙂 

```suggestion
import { ArgTypes, Title, Subtitle } from '@storybook/addon-docs';

<Title>Toast</Title>

<Subtitle>Usage</Subtitle>

<Subtitle>👍 When to use</Subtitle>

<Title>Properties</Title>

<ArgTypes of="bq-toast" />
```